lib/plaid/connection.rb

```diff
@@ -9,22 +9,29 @@ class << self
       def post(path, options = {})
         uri = build_uri(path)
         options.merge!(client_id: Plaid.customer_id, secret: Plaid.secret)
-        res = Net::HTTP.post_form(uri,options)
+        http = Net::HTTP.new(uri.host, uri.port)
+        http.use_ssl = (uri.scheme == 'https')
```

Comment:
Hi Nitesh , yep, I saw this. Generally your PR is fine, but I doubt anyone wants to send their bank transaction tokens via unencrypted connection, that's why I believe SSL should be `true` by default.

PS: By the way, I'm not Plaid developer, we just use Plaid in our startup...